awx/main/tests/functional/test_inventory_source_injectors.py

```diff
@@ -202,7 +202,7 @@ def test_inventory_update_injected_content(this_kind, inventory, fake_credential
     ExecutionEnvironment.objects.create(name='Default Job EE', managed=False)
 
     injector = InventorySource.injectors[this_kind]
-    if injector.plugin_name is None:
+    if injector.plugin_name in (None, 'constructed'):
```

Comment:
Could you explain why this needs to be skipped and why it wasn't before?

Reply:
Also, it might be good to filter out this case in the list of things passed to parametrize so that this test does not appear as always skipped.

Additionally, according to the output above, the case of `None` never happens so this check and a dynamic skip aren't actually needed.

Reply:
According to the Codecov coverage report, https://app.codecov.io/gh/ansible/awx/commit/764dcbf94b51f9b55b467824208e4c8b9dc15786/blob/awx/main/tests/functional/test_inventory_source_injectors.py#L206, it's never hit on `devel` either.

Reply:
This has been resolved via this commit: [705d8c081226f1dc36cf1d193e1a647d512b00dc](https://github.com/ansible/awx/pull/15537/commits/705d8c081226f1dc36cf1d193e1a647d512b00dc)